libs/sdk-backend-tiger/src/convertors/fromBackend/MetricConverter.ts

```diff
@@ -1,13 +1,27 @@
 // (C) 2021 GoodData Corporation
 
-import { JsonApiMetricOutDocument } from "@gooddata/api-client-tiger";
+import {
+    JsonApiMetricInAttributes,
+    JsonApiMetricOutDocument,
+    JsonApiMetricOutWithLinks,
+} from "@gooddata/api-client-tiger";
 import { newMeasureMetadataObject } from "@gooddata/sdk-backend-base";
 import { IMeasureMetadataObject } from "@gooddata/sdk-backend-spi";
 import { idRef } from "@gooddata/sdk-model";
 import { isInheritedObject } from "./utils";
 
-export function convertMetricFromBackend(data: JsonApiMetricOutDocument): IMeasureMetadataObject {
-    const { id, attributes } = data.data;
+export function convertMetricFromBackend(
+    data: JsonApiMetricOutDocument | JsonApiMetricOutWithLinks,
+): IMeasureMetadataObject {
+    let id: string;
+    let attributes: JsonApiMetricInAttributes;
+    if ("data" in data) {
```

Comment:
What is purpose of this?

Reply:
The data I receive has another type, so i attempted to reuse the method to get MeasureMetadata object from it

Reply:
Ok, so please let's not depend on "data" property, but let's create a type-guard for JsonApiMetricOutWithLinks type.